code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -388,6 +388,7 @@
 
 	return ..()
 
+
```

Comment:
```suggestion
```

Reply:
This is for readability, separating definitions of basetype closet procs and a subtype